### PR TITLE
Translation update to 1.3.66

### PR DESCRIPTION
**Pull request by JDR-Ninja**
I'm not sure for
 "FSC.DateFormat.YearOptions": "Options Année"
I hesitate with "**Mise en forme** Année" and "**Options** Année".

And also the word **padding**, I am not convinced of my text.

### CODE REVIEW

**Comment by vigoren** [comment]
Perhaps @TheBird956 could weigh in on those concerns?

Either way thanks so much for the translations!

File: src/lang/fr.json [hunk 1]
```diff
@@ -308,5 +304,98 @@
   "FSC.Module.AboutTime.Configuration.Message": "Nous avons détecté que le module About-Time est/a été activé pour ce monde.<br/><br/>Si vous souhaitez importer le calendrier de About-Time vers Simple Calendar, choisissez \"Importer vers Simple Calendar\", cela fera correspondre Simple Calendar aux paramètres de About-Time.<br/><br/>Si vous souhaitez que About-Time corresponde à Simple Calendar, choisissez \" Exporter vers About-Time\", cela fera correspondre les paramètres de About-Time à la configuration de Simple Calendar.<br/>",
   "FSC.Module.AboutTime.Export": "Exporter vers About-Time",
   "FSC.Importer.NoFile": "Veuillez sélectionner un fichier à importer.",
-  "FSC.Importer.InvalidSCConfig": "Fichier non valide téléchargé, veuillez télécharger un fichier JSON exporté par Simple Calendar."
-}
+  "FSC.Importer.InvalidSCConfig": "Fichier non valide téléchargé, veuillez télécharger un fichier JSON exporté par Simple Calendar.",
```
Comment (TheBird956) [inline src/lang/fr.json:307]: I would change this for "Le fichié téléversé est invalide, veuillez choisir un fichier JSON exporté par Simple Calendar"

Writing it this way is less redundant, and more accurate.

File: src/lang/fr.json [hunk 2]
```diff
@@ -308,5 +304,98 @@
...
+  "FSC.Midnight": "Minuit",
+  "FSC.Dawn": "Aube",
+  "FSC.Midday": "Midi",
+  "FSC.Dusk": "Crépuscule",
+  "FSC.DateFormat.Title": "Règles de mise en forme",
```
Comment (TheBird956) [inline src/lang/fr.json:312]: This should be "Symboles de mise en forme"

File: src/lang/fr.json [hunk 3]
```diff
@@ -308,5 +304,98 @@
...
+  "FSC.DateFormat.Description": "Une liste des différents symboles de mise en forme à utiliser dans la configuration de mise en forme ci-dessous.",
+  "FSC.DateFormat.YearOptions": "Options Année",
+  "FSC.DateFormat.MonthOptions": "Options Mois",
+  "FSC.DateFormat.DayOptions": "Options Jour",
+  "FSC.DateFormat.WeekdayOptions": "Options Jour de le semaine",
+  "FSC.DateFormat.HourOptions": "Options Heure",
+  "FSC.DateFormat.MinuteOptions": "Options Minute",
+  "FSC.DateFormat.SecondOptions": "Options Seconde",
```
Comment (TheBird956) [inline src/lang/fr.json:320]: Determinants are missing here

File: src/lang/fr.json [hunk 4]
```diff
@@ -308,5 +304,98 @@
...
+  "FSC.DateFormat.Year.NameExample": "Ral's Fury",
+  "FSC.DateFormat.Year.Prefix": "Préfix de l'année défini dans Préfix Année.",
+  "FSC.DateFormat.Year.PrefixExample": "Pre",
+  "FSC.DateFormat.Year.Postfix": "Suffix de l'année défini dans Suffix Année.",
+  "FSC.DateFormat.Year.PostfixExample": "AD",
+  "FSC.DateFormat.Month.Number": "Nombres des mois.",
+  "FSC.DateFormat.Month.NumberExample": "1 2 ... 11 12",
+  "FSC.DateFormat.Month.NumberPadded": "Nombres des mois remplis avec un zéro.",
+  "FSC.DateFormat.Month.NumberPaddedExample": "01 02 ... 11 12",
+  "FSC.DateFormat.Month.AbbreviatedName": "Les noms abrégé des mois défini dans Options Mois.",
```
Comment (TheBird956) [inline src/lang/fr.json:336]: Missing determinant when referring to the options

File: src/lang/fr.json [hunk 5]
```diff
@@ -308,5 +304,98 @@
...
+  "FSC.DateFormat.Month.AbbreviatedNameExample": "Jan Fév ... Nov Déc",
+  "FSC.DateFormat.Month.Name": "Les noms complet des mois défini dans Options Mois.",
```
Comment (TheBird956) [inline src/lang/fr.json:338]: Missing determinant when referring to the options

File: src/lang/fr.json [hunk 6]
```diff
@@ -308,5 +304,98 @@
...
+  "FSC.DateFormat.Month.NameExample": "Janvier Février ... Novembre Décembre",
+  "FSC.DateFormat.Day.Number": "Nombres des jours.",
+  "FSC.DateFormat.Day.NumberExample": "1 2 ... 30 31",
+  "FSC.DateFormat.Day.NumberPadded": "Nombres des jours remplis avec un zéro.",
+  "FSC.DateFormat.Day.NumberPaddedExample": "01 02 ... 30 31",
+  "FSC.DateFormat.Day.NumberSuffix": "Nombres des jours suivi de leurs suffix.",
+  "FSC.DateFormat.Day.NumberSuffixExample": "1er 2e ... 30e 31e",
+  "FSC.DateFormat.Weekday.Number": "Nombres des jours de la semaine.",
```
Comment (TheBird956) [inline src/lang/fr.json:346]: "Numéro du jour de la semaine"

File: src/lang/fr.json [hunk 7]
```diff
@@ -308,5 +304,98 @@
...
+  "FSC.DateFormat.Weekday.NumberExample": "1 2 ... 6 7",
+  "FSC.DateFormat.Weekday.NumberPadded": "Nombre des jours de la semaine rempli avec un zéro.",
+  "FSC.DateFormat.Weekday.NumberPaddedExample": "01 02 ... 06 07",
+  "FSC.DateFormat.Weekday.AbbreviatedName": "Nom de la semaine abrégé.",
```
Comment (TheBird956) [inline src/lang/fr.json:350]: "Nom abrégé de le semaine"

File: src/lang/fr.json [hunk 8]
```diff
@@ -308,5 +304,98 @@
...
+  "FSC.DateFormat.Weekday.AbbreviatedNameExample": "Dim Lun ... Ven Sam",
+  "FSC.DateFormat.Weekday.Name": "Noms complets des jours de la semaine.",
+  "FSC.DateFormat.Weekday.NameExample": "Dimanche Lundi ... Vendredi Samedi",
+  "FSC.DateFormat.Hour.12Hour": "Nombres des heures en format 12 heures",
+  "FSC.DateFormat.Hour.12HourExample": "1 2 ... 11 12",
+  "FSC.DateFormat.Hour.24Hour": "Nombres des heures en format 24 heures",
+  "FSC.DateFormat.Hour.24HourExample": "0 1 ... 22 23",
+  "FSC.DateFormat.Hour.12HourPadded": "Nombres des heures en format 12 heures remplis avec un zéro.",
```
Comment (TheBird956) [inline src/lang/fr.json:358]: "L'heure en format 12 heures remplis avec un zéro"

File: src/lang/fr.json [hunk 9]
```diff
@@ -308,5 +304,98 @@
...
+  "FSC.DateFormat.Weekday.NumberExample": "1 2 ... 6 7",
+  "FSC.DateFormat.Weekday.NumberPadded": "Nombre des jours de la semaine rempli avec un zéro.",
+  "FSC.DateFormat.Weekday.NumberPaddedExample": "01 02 ... 06 07",
+  "FSC.DateFormat.Weekday.AbbreviatedName": "Nom de la semaine abrégé.",
+  "FSC.DateFormat.Weekday.AbbreviatedNameExample": "Dim Lun ... Ven Sam",
+  "FSC.DateFormat.Weekday.Name": "Noms complets des jours de la semaine.",
+  "FSC.DateFormat.Weekday.NameExample": "Dimanche Lundi ... Vendredi Samedi",
+  "FSC.DateFormat.Hour.12Hour": "Nombres des heures en format 12 heures",
+  "FSC.DateFormat.Hour.12HourExample": "1 2 ... 11 12",
+  "FSC.DateFormat.Hour.24Hour": "Nombres des heures en format 24 heures",
```
Comment (TheBird956) [inline src/lang/fr.json:356]: "L'heure en format 24 heures"

File: src/lang/fr.json [hunk 10]
```diff
@@ -308,5 +304,98 @@
...
+  "FSC.DateFormat.Day.NumberSuffixExample": "1er 2e ... 30e 31e",
+  "FSC.DateFormat.Weekday.Number": "Nombres des jours de la semaine.",
+  "FSC.DateFormat.Weekday.NumberExample": "1 2 ... 6 7",
+  "FSC.DateFormat.Weekday.NumberPadded": "Nombre des jours de la semaine rempli avec un zéro.",
+  "FSC.DateFormat.Weekday.NumberPaddedExample": "01 02 ... 06 07",
+  "FSC.DateFormat.Weekday.AbbreviatedName": "Nom de la semaine abrégé.",
+  "FSC.DateFormat.Weekday.AbbreviatedNameExample": "Dim Lun ... Ven Sam",
+  "FSC.DateFormat.Weekday.Name": "Noms complets des jours de la semaine.",
+  "FSC.DateFormat.Weekday.NameExample": "Dimanche Lundi ... Vendredi Samedi",
+  "FSC.DateFormat.Hour.12Hour": "Nombres des heures en format 12 heures",
```
Comment (TheBird956) [inline src/lang/fr.json:354]: "L'heure en format 12 heures"

File: src/lang/fr.json [hunk 11]
```diff
@@ -308,5 +304,98 @@
...
+  "FSC.DateFormat.Hour.12HourExample": "1 2 ... 11 12",
+  "FSC.DateFormat.Hour.24Hour": "Nombres des heures en format 24 heures",
+  "FSC.DateFormat.Hour.24HourExample": "0 1 ... 22 23",
+  "FSC.DateFormat.Hour.12HourPadded": "Nombres des heures en format 12 heures remplis avec un zéro.",
+  "FSC.DateFormat.Hour.12HourPaddedExample": "01 02 ... 11 12",
+  "FSC.DateFormat.Hour.24HourPadded": "Nombres des heures en format 24 heures remplis avec un zéro.",
```
Comment (TheBird956) [inline src/lang/fr.json:360]: "L'heure en format 24 heures remplis avec un zéro"

File: src/lang/fr.json [hunk 12]
```diff
@@ -308,5 +304,98 @@
...
+  "FSC.Configuration.General.Display.DateFormat.DateHelp": "Le texte de mise en forme utilisé pour afficher une date en texte.",
+  "FSC.Configuration.General.Display.DateFormat.Time": "Mise en forme Heure",
+  "FSC.Configuration.General.Display.DateFormat.TimeHelp": "Le texte de mise en forme utilisé pour afficher une heure en texte.",
+  "FSC.Configuration.General.Display.DateFormat.MonthYear": "Mise en forme Mois/Année",
+  "FSC.Configuration.General.Display.DateFormat.MonthYearHelp": "Le texte de mise en forme utilisé pour afficher uniquement le mois et l'année qui apparaissent en haut dans la vue du calendrier.",
+  "FSC.Configuration.Season.StartingDate": "Date de départ",
+  "FSC.Configuration.Season.StartingDateHelp": "Le mois et le jour où cette saison commence chaque année.",
+  "FSC.Configuration.Season.SunriseSunset": "Heure du Lever/Coucher du soleil",
+  "FSC.Configuration.Season.SunriseSunsetHelp": "L'heure de la journée où le soleil se lève et se couche pour le premier jour de la saison.",
+  "FSC.Configuration.Month.Abbreviation": "Noms abrégés des Mois",
```
Comment (TheBird956) [inline src/lang/fr.json:395]: Unnecessary plurial

File: src/lang/fr.json [hunk 13]
```diff
@@ -308,5 +304,98 @@
...
+  "FSC.Configuration.Month.AbbreviationHelp": "Nom abrégé de ce mois.",
+  "FSC.Configuration.Weekday.Abbreviation": "Noms abrégés des Jours de la Semaine",
```
Comment (TheBird956) [inline src/lang/fr.json:397]: Unnecessary plurial

**Review by TheBird956 — CHANGES_REQUESTED** [review]

**Comment by TheBird956** [comment]
@JDR-Ninja  There isn't really a french translation to describe a "padded number". I looked at the PHP documentation (since I know it is translated into French) and they use the term "Nombre complété" (completed number) which doesn't really make sense to me, a developer, so I don't think it will make much sense for a non-developer. I think the closest way to say it would be "en nombre à deux chiffre", though it is a bit wordy.

For the options themselves, you could go with "Jetons d'année" (year tokens), regardless of what I said on the in-code review ;)

**Comment by JDR-Ninja** [comment]
I just commited changes.
